lib/endare_lints.yaml

```diff
@@ -22,6 +22,7 @@ linter:
     avoid_unused_constructor_parameters: false # Too opinionated.
     cancel_subscriptions: true
     close_sinks: true
+    comment_references: true # Helps with unresolved doc symbols.
```

Comment:
This lint helps with unresolved doc symbols.

I.e.

```dart
/// Something with [Bar]. // LINT on [Bar] if it is not in scope.
class Foo {}
```

The above can be fixed by adding an import (if Bar is used in actual code) _or_ with the new `@docImport` annotation on a library element:

```dart
/// @docImport './bar.dart';
library;

/// Something with [Bar]. // no lint, [Bar] is in scope for documentation, due to the doc import. [Bar] is still not in scope for actual code, though.
class Foo {}
```